packages/graphiql/src/components/VariableEditor.tsx

```diff
@@ -11,7 +11,15 @@ import React from 'react';
 
 import onHasCompletion from '../utility/onHasCompletion';
 import commonKeys from '../utility/commonKeys';
-import { SizerComponent } from 'src/utility/CodeMirrorSizer';
+
+declare module CodeMirror {
```

Comment:
I was trying to fix this before, so I opened a PR in DefinitelyTyped : https://github.com/DefinitelyTyped/DefinitelyTyped/pull/42194 I think this can be removed when that gets merged?

Reply:
great! i was too lazy to do that :laughing: 

monaco is typescript native and will be much easier, so all of the codemirror stuff will be vanishing soon